R/install_easystats.R

```diff
@@ -7,6 +7,8 @@ easystats_zen <- function(){
 
 #' Install the easystats suite from github
 #'
+#' This function can be used to install all the easystats package from github, either from the master branch (the stable one) with \code{install_easystats_latest} or from the dev branch \code{install_easystats_dev}.
```

Comment:
Just some minor stylistic issue, I would add parantheses to `\code{install_easystats_latest}` -> `\code{install_easystats_latest()}`, and to the second `\code{}` tag as well.

Reply:
@zen-juen 👀

Reply:
Just done!

Reply:
A true master githubber!